app/adapters/application.js

```diff
@@ -1,3 +0,0 @@
-import LFAdapter from 'ember-localforage-adapter/adapters/localforage';
```

Comment:
I do not get this. But perhaps I should first investigate some time with ember-cli and ember-addons.